spec/function.dd

```diff
@@ -2884,10 +2884,60 @@ $(H3 $(LNAME2 safe-functions, Safe Functions))
         so they can be corrected.
         )
 
+$(H4 Safe External Functions)
+
+        $(P External functions don't have a function body visible to the compiler:
+        ---
+        @safe extern (C) void play();
+        ---
+        and so safety cannot be verified automatically.
+        )
+
+        $(BEST_PRACTICE Explicitly set an attribute for external functions rather
+        than relying on default settings.)
```

Comment:
A "best practice" doesn't absolve DIP 1028 silently changing the default for the worse.

Reply:
This is not about DIP 1028.

Reply:
> This is not about DIP 1028.

Sure.